### PR TITLE
admin-panel

### PR DESCRIPTION
**Pull request by ScottMktn**

### CODE REVIEW

**Comment by ScottMktn** [comment]
Screenshot 2024-05-20 at 3 28 56 PM

**Comment by ScottMktn** [comment]
Screenshot 2024-05-20 at 3 29 16 PM

File: valhalla/jawn/src/controllers/private/adminController.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            id: org.id,
+          })) ?? [],
+      };
+    } else {
+      throw new Error("Unauthorized");
+    }
+  }
+
+  @Post("/admins/org/query")
+  public async addAdminsToOrg(
```
Comment (chitalian) [inline valhalla/jawn/src/controllers/private/adminController.ts:78]: Please add the admin check here.

File: supabase/migrations/20240521180156_admin.sql [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+using (true);
+
+
+create policy "Enable update access for all users"
+on "public"."alert_banners"
+as permissive
+for update
+to public
+using (true)
+with check (true);
```
Comment (chitalian) [inline supabase/migrations/20240521180156_admin.sql:145]:
```suggestion
alter table "public"."admins" validate constraint "admins_user_id_fkey";

create policy "Enable read access for all users"
on "public"."alert_banners"
as permissive
for select
to public
using (true);
```

**Pull request merged**